cgs_vmc/utils.py

```diff
@@ -83,6 +83,8 @@ def create_hparams(**kwargs: Any) -> tf.contrib.training.HParams:
       basis_file_path='',
 
       wavefunction_type='',
+      wavefunction_type1='',
```

Comment:
Naming parameters with indices is a bad idea, as it is almost impossible to tell what they represents.
You should also add documentation to the docstring above.

We could use something like `sum_wavefunction_types=('', '')` i.e. a tuple of strings, or more generic `composit_wavefunction_types`.